.travis.yml

```diff
@@ -1,15 +0,0 @@
-language: go
```

Comment:
why? And why in this PR?

Reply:
I have setup proper CI at the same time and requiring changes in this PR

Reply:
that also implies we should merge this soon :)